config/routes.rb

```diff
@@ -1,67 +1,16 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  root 'people#index'
 
-  # You can have the root of your site routed with "root"
-  root 'tasks#index'
-  put '/tasks/complete/:id' => 'tasks#complete'
+  get '/tasks' => 'tasks#all_tasks'
 
-  resources :tasks
-    # HTTPverb, Path, Controller#Action, Used for
-    # GET /tasks tasks#index  display a list of all tasks
-    # GET /tasks/new tasks#new  return an HTML form for creating a new task
-    # POST /tasks tasks#create create a new task
-    # GET /tasks/:id tasks#show display a specific task
-    # GET /tasks/:id/edit  tasks#edit return an HTML form for editing a task
-    # PATCH/PUT /tasks/:id tasks#update update a specific task
-    # DELETE  /tasks/:id tasks#destroy  delete a specific task
+  resources :people do
+    resources :tasks, only: [:index, :new, :create, :update]
+  end
+  resources :tasks, only: [:show, :edit, :destroy]
 
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
+  # # NOTE: this does not work because :create needs to be a nested action in order for the edit form to submit correctly.
```

Comment:
So, I'm not 100% here, but some limited testing on my part makes me think that using the `member` block for nesting may have done the trick. We can try it out sometime if you like.

```ruby
resources :people do
  member do
    resources :tasks, shallow: true
  end
end
```